cypress/integration/Change Password Tests/changePassword.specs.js

```diff
@@ -0,0 +1,43 @@
+ /// <reference types="cypress" />
+ import {passwordTab} from  '../../utils/changePasswordTab'
```

Comment:
Please fix the formatting.
```suggestion
 import { passwordTab } from  '../../utils/changePasswordTab';
 
```